CDCgov/trusted-intermediary
# Bug Fix - Support for Windows and Unix-base Systems End Of Line Characters

# Clean Keys from End of Line Characters

 This PR fixes a new bug that was detected when using docker to run our application in Windows. The bug was related to the parsing of our public key. When Running TI in docker, in a Windows environment, the key had `\r\n` characters do to the way Windows end of line characters work.

## Solution
Normalize key from `\r\n to \n` and then clean the key from `\n`.

## How to Test
- run `./gradlew shadowJar`
- run `docker compose up --build` to spin up the containers
- request an access token to our auth endpoint
- make sure request is successful
- stop containers by running `docker compose down`
- run `./gradlew :app:test && ./gradlew :shared:test && ./gradlew :etor:test && ./e2e-execute.sh`

## Issue
#1244 

## Checklist
